File: bes/bes3bak/etaprime/data/cutmassroot/tightcut_zhou2me/a.c

void a()
{
    TChain c("tree");
    c.Add("data.root");

    TH1F h1("h1", "h1", 20, 2.25, 2.35);
    c.Project("h1", "M_BC_r3c", "M_BC_r3c>2.25 && np==npbar && deltaE_min>-0.02 && deltaE_min<0.015"); h1.Draw();

    c.Project("h1", "M_BC_r3c", "M_BC_r3c>2.25 && chi2_min_r3c<10"); h1.Draw();
    c.Project("h1", "M_BC_r3c", "M_BC_r3c>2.25 && && np!=0 && npbar!=0 && chi2_min_r3c<29"); h1.Draw();
}

File: bes/bes3bak/etaprime/data/cutmassroot/tightcut_zhou2me/a.py
import ROOT
from ROOT import * 

f=["4600data.root", "4600data_removedouble.root", 
    "4612data.root", "4612data_removedouble.root", 
    "4620data.root", "4620data_removedouble.root", 
    "4640data.root", "4640data_removedouble.root",
    "4660data.root","4660data_removedouble.root",
    "4680data.root","4680data_removedouble.root",
    "4700data.root","4700data_removedouble.root" ]

c=TChain ("tree")

for i in range(len(f)):
    c.Add(f[i])
    print f[i], c.GetEntries()
    c.Reset()



